feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -275,7 +275,7 @@ class BillingWrapperTest {
                 fail("shouldn't be an error")
             })
 
-        assertThat(numCallbacks == 1)
+        assert(numCallbacks == 1)
```

Comment:
Hmm I think the pattern we've been following is
```suggestion
        assertThat(numCallbacks).isEqualTo(1)
```